tests/e2e/specs/frontend_tests.js

```diff
@@ -0,0 +1,192 @@
+// https://docs.cypress.io/api/introduction/api.html
```

Comment:
These tests are fantastic!!!!! A few of them will have to be modified once we begin working with the live data